app/scripts/modules/core/src/pipeline/config/stages/monitoreddeploy/DeploymentMonitorExecutionDetails.tsx

```diff
@@ -0,0 +1,116 @@
+  const { context } = stage;
+
+  return get(context, 'deploymentMonitorReasons.summary', '<NOT PROVIDED>');
+}
+
+function getDeploymentMonitorName(stage: IExecutionStage, monitors: IDeploymentMonitorDefinition[]) {
+  return monitors.find(x => x.id === stage.context.deploymentMonitor.id).name;
+}
+
+function getDeploymentMonitorSupportUrl(stage: IExecutionStage, monitors: IDeploymentMonitorDefinition[]) {
+  return monitors.find(x => x.id === stage.context.deploymentMonitor.id).supportContact;
+}
+
+function getDeploymentMonitorDetails(stage: IExecutionStage) {
+  const { context } = stage;
+
+  return get(context, 'deploymentMonitorReasons.reason.message', '<NOT PROVIDED>');
+}
+
+function getDeploymentMonitorLog(stage: IExecutionStage) {
+  const { context } = stage;
+  const logArray = get(context, 'deploymentMonitorReasons.reason.logSummary', ['<NOT PROVIDED>']);
+
+  return logArray.join('\n');
+}
+
+function getDeploymentMonitorAdditionalDetails(stage: IExecutionStage) {
+  const { context } = stage;
+  const additionalData = get(context, 'deploymentMonitorReasons.reason.additionalData', []) as IAdditionalData[];
+
+  if (!isEmpty(additionalData)) {
+    return additionalData.map(({ key, link, text }) => (
+      <React.Fragment key={key + link + text}>
+        <dt>{key}</dt>
+        <dd>
+          <a href={link} target="_blank">
+            {text}
+          </a>
+        </dd>
+      </React.Fragment>
+    ));
+  }
+
+  return null;
+}
+
+interface IDeploymentMonitorExecutionDetailsSectionState {
+  deploymentMonitors: IDeploymentMonitorDefinition[];
+}
+
+export class DeploymentMonitorExecutionDetails extends React.Component<
+  IExecutionDetailsSectionProps,
+  IDeploymentMonitorExecutionDetailsSectionState
+> {
+  public static title = 'evaluateDeploymentHealth';
+  public state: IDeploymentMonitorExecutionDetailsSectionState = { deploymentMonitors: null };
+
+  public componentDidMount(): void {
+    DeploymentMonitorReader.getDeploymentMonitors().then(deploymentMonitors => {
+      this.setState({ deploymentMonitors });
+    });
+  }
+
+  public render() {
+    const { stage, current, name } = this.props;
+    const { deploymentMonitors } = this.state;
+
+    return (
+      <ExecutionDetailsSection name={name} current={current}>
+        <div className="row">
+          <div className="col-md-12">
+            <dl className="dl-narrow dl-horizontal">
+              <dt>Summary</dt>
+              <dd>{getDeploymentMonitorSummary(stage)}</dd>
+              <dt>Details</dt>
+              <dd>{getDeploymentMonitorDetails(stage)}</dd>
+              <dt className="sp-margin-l-bottom">Log</dt>
+              <dd className="sp-margin-l-bottom">{getDeploymentMonitorLog(stage)}</dd>
+              {getDeploymentMonitorAdditionalDetails(stage)}
+            </dl>
+            {deploymentMonitors && (
+              <div>
+                <strong>NOTE:</strong> This information is provided by the{' '}
```

Comment:
is the `{' '}` necessary here with the `<i>` tag following?

Reply:
i was trying to put a space, and it was giving me a hard time, i removed it and replaced with &nbsp;